nielsendtvr/src/main/java/com/segment/analytics/android/integrations/nielsendtvr/NielsenDTVRIntegrationFactory.java

```diff
@@ -0,0 +1,157 @@
+package com.segment.analytics.android.integrations.nielsendtvr;
+
+import android.content.Context;
+
+import com.segment.analytics.Analytics;
+import com.segment.analytics.ValueMap;
+import com.segment.analytics.integrations.Integration;
+import com.segment.analytics.integrations.Logger;
+import com.nielsen.app.sdk.AppSdk;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.segment.analytics.internal.Utils.isNullOrEmpty;
+
+class NielsenDTVRIntegrationFactory implements Integration.Factory {
+
+    private static final String NIELSEN_DTVR_KEY = "Nielsen DTVR";
+    private static final Map<String, List<AppSdk>> appSdkInstances = new HashMap<>();
+    private static final int MAX_INSTANCES = 4;
+
+    static final String SETTING_APP_ID_KEY = "appId";
+    static final String SETTING_SF_CODE_KEY = "sfcode";
+    static final String SETTING_ID3_EVENTS_KEY = "sendId3Events";
+    static final String SETTING_ID3_PROPERTY_KEY = "id3Property";
+    static final String SETTING_ID3_PROPERTY_DEFAULT = "Id3";
```

Comment:
let's call it `id3` :)

Reply:
(Sorry, it was my bad, I'll correct that in my previous comment. The default field name should be `id3`).

Reply:
Hi there, we were previously given the `Id3` value to use as a default - we're good to change it, and we'll keep iOS in sync. Let us know if we're going with `id3` or `Id3`

Reply:
Sorry about that @davidhazama . Could we use `id3` for iOS and Android? (all lowercase)

Reply:
No worries, will do! :+1: